vagrant-spk

```diff
@@ -511,6 +511,14 @@ def setup_vm(args):
     with open(vagrantfile_path, "w") as f:
         f.write(VAGRANTFILE_CONTENTS)
 
+    # Recursively copy in anything in the stack's service-config folder.
```

Comment:
Since we're recursively copying, doing symlinks for `mime.types` should work OK. (But maybe not on Windows. But I can make the Windows installer do the copying, and we build the Windows installer on Linux anyway.)